### ISSUE TITLE
Unrecognized field name "dynamicOpts"

### ISSUE THREAD

**Issue by RobinKlement** [scipopt/MatlabSCIPInterface#6]
Unfortunately, after the update from 3 weeks ago the first example from "MixedInteger_LinearProgramming.m" does not run anymore. Before the update it worked.

I get the error message

```
Unrecognized field name "dynamicOpts".

Error in opti/displayOPTI (line 356)
dopts = optidynset(opts.dynamicOpts);

Error in opti/display (line 233)
                displayOPTI(optObj);

Error in MixedInteger_LinearProgramming (line 29)
Opt = opti('f',f,'ineq',A,b,'bounds',lb,ub,'xtype',xtype,'options',opts)
```

Disclaimer: I have updated MATLABScipInterface on my machine and recompiled the mex file. The new mex file seems to be broken and Matlab freezes, leaving me with the only option of force-quitting Matlab. The same happens with the post-installation tests. Therefore I started it with the old mex file.

[opti_debug.txt](https://github.com/scipopt/MatlabSCIPInterface/files/11286118/opti_debug.txt)

**Comment by pfetsch** [comment]
Indeed, I have overlooked this part. I have fixed (removed) this part. Also the MixedInteger_LinearProgramming examples should now run trough. I have also extended the list by one example the uses solver specific options at the end.

**Comment by RobinKlement** [comment]
Thank you, the examples work again.

Disclaimer: I have updated MATLABScipInterface on my machine and recompiled the mex file. When calling the newly compiled mex file via "scip", Matlab freezes, leaving me with the only option of force-quitting Matlab. The same happens with the post-installation tests. Therefore I have only tested with the old mex file.

[opti_debug.txt](https://github.com/scipopt/MatlabSCIPInterface/files/11677499/opti_debug.txt)
